src/actions/leaderBoardData.js

```diff
@@ -15,6 +16,15 @@ export const getLeaderboardData = userId => {
   };
 };
 
```

Comment:
Error handling will be good choice to help debug

```suggestion
export const getLeaderboardData = userId => async dispatch => {
  try {
    const url = ENDPOINTS.LEADER_BOARD(userId);
    const response = await httpService.get(url);
    dispatch(getLeaderboardDataActionCreator(response.data));
  } catch (error) {
    console.error('Failed to fetch leaderboard data:', error);
  }
};

```